spec/services/activity/import_spec.rb

```diff
@@ -320,9 +320,9 @@
       expect(existing_activity.objectives).to eq(existing_activity_attributes["Aims/Objectives"])
       expect(existing_activity.beis_identifier).to eq(existing_activity_attributes["BEIS ID"])
       expect(existing_activity.uk_po_named_contact).to eq(existing_activity_attributes["UK PO Named Contact"])
-      expect(existing_activity.sdgs_apply).to eql(true)
+      expect(existing_activity.sdgs_apply).to eq(true)
```

Comment:
I think we should use `be(true)` or `be_true`, because Ruby Booleans are singletons

Reply:
This is old but I think `be_true` is equivalent to `be_truthy`: https://bengribaudo.com/blog/2014/01/17/2674/rspec-be-true-be_true-are-different

We use `eq(true)` or `eq true` more than `be true` by the looks of it. Re: singletons, is the distinction that `be` suggests it is the single value `true` whereas `eq` suggests equivalence between two `true`s?

Happy to change it if `be` makes more sense!

Reply:
Will update later!